Controls/Cafe/Customer1.c

```diff
@@ -0,0 +1,29 @@
+#include "Cafe.h"
+
+/**
+ * Customer 1 wants to buy something!
+ */
+
+/**
+ * @brief Get the FOURTH item and cost on the menu (Cafe.h). Return the name and cost
+ * of the item to the pointers passed into the function. Add the cost to the business 
+ * revenue and print the current business revenue.
+ */
+void Customer1_checkout(char* name, int* cost) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
+/**
+ * @brief Acquire the register. Call checkout. Print the name and cost of the item.
```

Comment:
It feels weird to acquire the mutex and then call checkout, which is the actual place where the modification is made. In general, we want to ensure the time we hold a mutex is as short as possible. Since the mutex is only guarding the revenue variable, the mutex should be acquired, the value modified, and then the mutex released again.

Reply:
I guess in the context of the code the mutex only guards the revenue variable so it makes sense to surround only that modification with the acquire and release. The intention was just to make the mutex guard an abstract "register" which obviously shouldn't be accessed by both customers at the same time, so even purchasing an item could only be done by one customer at a time. Technically printing out the thing you're purchasing and the price doesn't require a mutex, but in real life you wouldn't be able to tell the cashier two different orders at the same time.

Reply:
Fair enough, but it might be worth trying to explain that in the README or the comments somehow.